activestorage/test/models/attached/many_test.rb

```diff
@@ -445,12 +445,15 @@ class ActiveStorage::ManyAttachedTest < ActiveSupport::TestCase
       @user.highlights.attach blobs
       assert @user.highlights.attached?
 
+      updated_at_before_purge = @user.updated_at
+
       @user.highlights.purge
```

Comment:
How about using `assert_changes` here?

```suggestion
      assert_changes -> { @user.updated_at } do
        @user.highlights.purge
      end
```

Reply:
@eugeneius Changed done on all tests, `assert_changes` looks easier to read 👍